index.html

```diff
@@ -7,26 +7,59 @@
     <link rel="stylesheet" href="./styles/styles.css" />
   </head>
   <body>
-    <div class="container">
+    <div class="container-advantage">
       <h1 class="title">What do we offer</h1>
+      <hr class="highlight" />
       <div class="row advantage">
-          <div class="col advantage-col">
-          <img class="advantage-img" src="./images/support.png" alt="support">
+        <div class="col advantage-col">
+          <img class="advantage-img" src="./images/support.png" alt="support" />
           <h2 class="advantage-title">Support 24/7</h2>
         </div>
         <div class="col advantage-col">
-          <img class="advantage-img" src="./images/communication.png" alt="communication">
+          <img
+            class="advantage-img"
+            src="./images/communication.png"
+            alt="communication"
+          />
           <h2 class="advantage-title">Communicate directly</h2>
         </div>
         <div class="col advantage-col">
-          <img class="advantage-img" src="./images/booking.png" alt="booking">
+          <img class="advantage-img" src="./images/booking.png" alt="booking" />
           <h2 class="advantage-title">Book online</h2>
         </div>
         <div class="col advantage-col">
-          <img class="advantage-img" src="./images/reviews.png" alt="reviews">
+          <img class="advantage-img" src="./images/reviews.png" alt="reviews" />
           <h2 class="advantage-title">Real guest reviews</h2>
         </div>
       </div>
     </div>
+    <div class="container-footer">
+      <div class="row footer">
+        <div class="col footer-col">
+          <h3 class="text">About</h3>
```

Comment:
1. It is not a header.
2. Be careful with its order if you use headers (`h3` < `h2`).

Headers in HTML are used by screen readers and other devices to build a table of content.